ember-primitives/src/components/accordion/header.gts

```diff
@@ -0,0 +1,43 @@
+import { hash } from '@ember/helper';
+
+import { getDataState } from './item';
+import Trigger from './trigger';
+
+import type { TOC } from '@ember/component/template-only';
+import type { WithBoundArgs } from '@glint/template';
+
+export const AccordionHeader: TOC<{
+  Element: HTMLDivElement;
+  Blocks: {
+    default: [{
+      /**
+       * The AccordionTrigger component.
+       */
+      Trigger: WithBoundArgs<typeof Trigger, 'value' | 'isExpanded' | 'disabled' | 'toggleItem'>;
+    }];
+  };
+  Args: {
+    /**
+     * The value of the accordion item.
+     */
+    value: string;
+    /**
+     * Whether the accordion item is expanded.
+     */
+    isExpanded: boolean;
+    /**
+     * Whether the accordion item is disabled.
+     */
+    disabled?: boolean;
+    /**
+     * A callback that is called when the accordion item is toggled.
+     */
+    toggleItem: () => void;
+  }
+}> = <template>
+  <div role='heading' aria-level='3' data-state={{getDataState @isExpanded}} data-disabled={{@disabled}} ...attributes>
```

Comment:
why was 3 chosen?

we should probably link to: https://www.w3.org/WAI/ARIA/apg/patterns/accordion/ on the docs, because, until: https://github.com/universal-ember/ember-primitives/issues/37
it's a user-concern to configure the heading level.

Reply:
Good point, I don't have a good reason why I went for 3. Do you think it's worth keeping it as a default but writing in the docs that this should be configured?

Reply:
Probably fine for now, yeah. Docs update would be great!